ZendeskClientApp/assets/navbar.js

```diff
@@ -0,0 +1,29 @@
+
+
+const setupSearch = async function() {
+
+    const settings = await getSettings();
+
+    // Initialize a Router
+    // See https://www.algolia.com/doc/api-reference/widgets/history-router/js/
+    const historyRouter = instantsearch.routers.history();
+
+    
+    const searchClient = algoliasearch(settings.algoliaAppName, settings.searchApiKey);
```

Comment:
`algoliaAppName` is not a good name if it's actually an AppID

Reply:
thanks..I agree.